google/cloud/aiplatform/utils.py

```diff
@@ -194,7 +194,21 @@ def validate_project(project: str) -> bool:
     raise ValueError("Please provide a valid project ID")
 
 
-def validate_region(region: str):
+def validate_display_name(display_name: str):
+    """Verify display name is at most 128 chars and UTF-8 only.
```

Comment:
How does this method validate `UTF-8`? If not, I think it's OK to capture in buganizer and TODO here.